common/rpc/grpc.go

```diff
@@ -181,20 +185,51 @@ func NewServiceErrorInterceptor(
 		}
 		return resp, serviceerror.ToStatus(err).Err()
 	}
+}
+
+func NewFrontendErrorInterceptor(
```

Comment:
I am against this name. I believe interceptors should be small and specific. This one should be called `ErrorMaskerInterceptor` or `ErrorHiderInterceptor`.

Reply:
changed to MaskInternalErrorsInterceptor